ee/clickhouse/models/property.py

```diff
@@ -102,7 +99,7 @@ def parse_prop_clauses(
                 allow_denormalized_props=allow_denormalized_props,
             )
 
-            final.append(f"{filter_query} AND {table_name}team_id = %(team_id)s" if team_id else filter_query)
+            final.append(f" {filter_query}")
```

Comment:
Nit: The space isn't needed based off of my reading of this.

Reply:
Hm I would agree with you but some tests did fail when I removed it.